src/xasl/access_json_table.cpp

```diff
@@ -353,7 +323,7 @@ namespace cubxasl
 	    }
 	  else if (m_expand_type == json_table_expand_type::JSON_TABLE_OBJECT_EXPAND)
 	    {
-	      m_iterator = db_json_create_iterator (DB_JSON_TYPE::DB_JSON_OBJECT);
+	      assert (false);
```

Comment:
remove all code related to JSON_TABLE_OBJECT_EXPAND. there is only expand or no expand.